### PR TITLE
48852 _cron_job_using_whenever

### PR DESCRIPTION
**Pull request by manazirahsan**

### CODE REVIEW

**Comment by noor-framgia** [comment]
ok

File: Gemfile [hunk 1]
```diff
@@ -29,14 +29,13 @@ gem "sinatra", require: false
 gem "slim"
 gem "redis"
 gem "public_activity", "~> 1.4.0"
+gem "whenever", :require => false
```
Comment (mdmahmudurrahman) [inline Gemfile]: Rails convention!

File: config/deploy.rb [hunk 2]
```diff
@@ -1 +1,5 @@
-require "sidekiq/capistrano"
\ No newline at end of file
+require "sidekiq/capistrano"
+require "whenever/capistrano"
+
+set :whenever_environment, defer { stage }
```
Comment (mdmahmudurrahman) [inline config/deploy.rb]: Rails convention!

**Comment by mdmahmudurrahman** [comment]
OK

**Comment by noor-framgia** [comment]
ok

**Pull request merged**